src/normalize.js

```diff
@@ -258,24 +264,98 @@ export const createNodes = (entity, ctx, uid) => {
   return nodes;
 };
 
-/**
- * Extract images and create remote nodes for images in all fields.
- * @param {Object} item the entity
- * @param {Object} ctx gatsby function
- * @param {String} uid the main schema uid
- */
-const extractImages = async (item, ctx, uid) => {
+const extractFiles = (text, apiURL) => {
+  const files = [];
+  // parse the markdown content
+  const parsed = reader.parse(text);
+  const walker = parsed.walker();
+  let event, node;
+
+  while ((event = walker.next())) {
+    node = event.node;
```

Comment:
Not sure I completely understand what's going on here but seems a bit strange to be checking for a value and assigning a value at the same time

```suggestion
  while (walker.next()) {
    event = walker.next()
    node = event.node;
```

Reply:
According to the [documentation](https://www.npmjs.com/package/commonmark) it is the correct way to use it.